cmd/gds/main.go

```diff
@@ -1420,20 +1420,29 @@ func loadProfile(c *cli.Context) (err error) {
 
 // helper function to create the GRPC client with default options
 func initClient(c *cli.Context) (err error) {
+	if profile.Directory == nil {
+		return cli.Exit("current profile does not contain directory configuration", 1)
+	}
 	if client, err = profile.Directory.Connect(); err != nil {
 		return cli.Exit(err, 1)
 	}
 	return nil
 }
 
 func initAdminClient(c *cli.Context) (err error) {
+	if profile.Admin == nil {
+		return cli.Exit("current profile does not contain admin configuration", 1)
+	}
 	if adminClient, err = profile.Admin.Connect(); err != nil {
 		return cli.Exit(err, 1)
 	}
 	return nil
 }
 
 func initMembersClient(c *cli.Context) (err error) {
+	if profile.Members == nil {
+		return cli.Exit("current profile does not contain members configuration", 1)
+	}
```

Comment:
These were an unrelated panic - but figured I'd fix the panic with a reasonable error message to make debugging easier in the future.